pom.xml

```diff
@@ -547,10 +547,11 @@
     </dependency>
 
     <!-- Security update. See https://broadworkbench.atlassian.net/browse/DUOS-2826 -->
+    <!-- Logback Access module moved to new location. See https://logback.qos.ch/news.html -->
     <dependency>
-        <groupId>ch.qos.logback</groupId>
-        <artifactId>logback-access</artifactId>
-        <version>${logback.version}</version>
+        <groupId>ch.qos.logback.access</groupId>
+        <artifactId>common</artifactId>
+        <version>2.0.3</version>
```

Comment:
How come this is hardcoded now, rather than referencing the variable?

Reply:
The versions are different now

Reply:
Ah, okay, I see. So the specific module is referenced separately. That's really frustrating.

Reply:
Yeah, it's a very non-standard way to do things - not sure why they took this approach.